src/lang/KclSingleton.ts

```diff
@@ -396,10 +396,8 @@ export class KclManager {
     // updateArtifactGraph relies on updated executeState/variables
     this.engineCommandManager.updateArtifactGraph(execState.artifactGraph)
     this._executeCallback()
-    if (!isInterrupted) {
+    if (!isInterrupted)
       sceneInfra.modelingSend({ type: 'code edit during sketch' })
-    }
-
```

Comment:
If this is just a formatter change I think we should discuss adding a prettier config to add the braces, I think they decrease ambiguity especially in cases like this where the expression is on the next line because it's so long.

Reply:
c1db093e0f74257e8d5d950df41da0d1ea07cf65